files/zh-cn/learn/javascript/first_steps/what_is_javascript/index.md

```diff
@@ -46,144 +46,88 @@ p {
   letter-spacing: 1px;
   text-transform: uppercase;
   text-align: center;
-  border: 2px solid rgba(0, 0, 200, 0.6);
-  background: rgba(0, 0, 200, 0.3);
-  color: rgba(0, 0, 200, 0.6);
-  box-shadow: 1px 1px 2px rgba(0, 0, 200, 0.4);
+  border: 2px solid rgb(0 0 200 / 0.6);
+  background: rgb(0 0 200 / 0.6);
+  color: rgb(255 255 255 / 1);
+  box-shadow: 1px 1px 2px rgb(0 0 200 / 0.4);
   border-radius: 10px;
   padding: 3px 10px;
   display: inline-block;
   cursor: pointer;
 }
 ```
 
-```html hidden
-<!DOCTYPE html>
-<html><head><style>
-p {
-  font-family: sans-serif, '黑体';
-  letter-spacing: 1px;
-  text-transform: uppercase;
-  text-align: center;
-  border: 2px solid rgba(0, 0, 200, 0.6);
-  background: rgba(0, 0, 200, 0.3);
-  color: rgba(0, 0, 200, 0.6);
-  box-shadow: 1px 1px 2px rgba(0, 0, 200, 0.4);
-  border-radius: 10px;
-  padding: 3px 10px;
-  display: inline-block;
-  cursor: pointer;
-}
-</style></head>
-<body>
-<p>玩家 1：小明</p>
-</body></html>
-```
-
-{{ EmbedLiveSample('广义的定义', '100%', 80) }}
+![已添加样式的“玩家 1：小明”段落](html-and-css.png)
 
 最后，我们可以再加上一些 JavaScript 来实现动态行为：
 
 ```js
-const para = document.querySelector('p');
+const para = document.querySelector("p");
 
-para.addEventListener('click', updateName);
+para.addEventListener("click", updateName);
 
-function updateName() {
-  let name = prompt('输入一个新的名字：');
-  para.textContent = '玩家 1：' + name;
-}
-```
-
-```html hidden
-<!DOCTYPE html>
-<html><head><style>
-p {
-font-family: sans-serif, '黑体';
-letter-spacing: 1px;
-text-transform: uppercase;
-text-align: center;
-border: 2px solid rgba(0,0,200,0.6);
-background: rgba(0,0,200,0.3);
-color: rgba(0,0,200,0.6);
-box-shadow: 1px 1px 2px rgba(0,0,200,0.4);
-border-radius: 10px;
-padding: 3px 10px;
-display: inline-block;
-cursor: pointer;
-}
-</style></head>
-<body>
-<p>玩家 1：小明</p>
-<script>
-const para = document.querySelector('p');
-para.addEventListener('click', updateName);
 function updateName() {
   const name = prompt('输入一个新的名字：');
-  para.textContent = '玩家 1：' + name;
+  para.textContent = `玩家 1：${name}`;
 }
-</script>
-</body></html>
 ```
 
-{{ EmbedLiveSample('广义的定义', '100%', 80) }}
+{{ EmbedLiveSample('高层定义', '100%', 80) }}
 
-尝试点击最后一个版本的文本标签，观察会发生什么（也可在 GitHub 上查看这个 demo 的 [源代码](https://github.com/roy-tian/learning-area/blob/master/javascript/introduction-to-js-1/what-is-js/javascript-label.html) 或 [在线运行](https://roy-tian.github.io/learning-area/javascript/introduction-to-js-1/what-is-js/javascript-label.html)。）
+尝试点击最后一个版本的文本标签，观察会发生什么（也可在 GitHub 上查看这个 demo 的 [源代码](https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/what-is-js/javascript-label.html)或[在线运行它](https://mdn.github.io/learning-area/javascript/introduction-to-js-1/what-is-js/javascript-label.html)）！
```

Comment:
Unfortunately the source code is not localized, probably it would be more appropriate to leave the above content untranslated.

```suggestion
尝试点击最后一个版本的文本标签，观察会发生什么（也可在 GitHub 上查看这个演示的[源代码](https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/what-is-js/javascript-label.html)或[在线运行它](https://mdn.github.io/learning-area/javascript/introduction-to-js-1/what-is-js/javascript-label.html)）！
```

Reply:
```suggestion
尝试点击最后一个版本的文本标签，观察会发生什么（也可在 GitHub 上查看这个演示的[源代码](https://github.com/roy-tian/learning-area/blob/master/javascript/introduction-to-js-1/what-is-js/javascript-label.html)或[在线运行它](https://roy-tian.github.io/learning-area/javascript/introduction-to-js-1/what-is-js/javascript-label.html)）！
```